src/Microsoft.DotNet.Darc/DarcLib/Helpers/ProcessManager.cs

```diff
@@ -45,13 +45,13 @@ public class ProcessManager : IProcessManager
 
     public string GitExecutable { get; }
 
-    public ProcessManager(ILogger<IProcessManager> logger, string gitExecutable)
```

Comment:
Why not ILogger<ProcessManager>?

Reply:
We do not use DI in the whole DarcLib and in many places we need to keep passing dependencies always:
https://github.com/dotnet/arcade-services/blob/cf18f0f7761dbc9f86fd8e33b58207200d8cacec/src/Microsoft.DotNet.Darc/DarcLib/Models/Darc/DependencyGraph.cs#L168-L174

In those places, we don't have the `ILoggerFactory` but `ILogger` only so it's just easier to pass it down the road like this.